### PR TITLE
fix: remove invalid `ctx` import from hello-world experimental templates

### PR DESCRIPTION
**Pull request by petebacondarwin**
## What this PR solves / how to test

Fixes #0000

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [x] Tests included
  - [ ] Tests not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required
  - [x] Not required because: test change
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Changeset included
  - [ ] Changeset not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Documentation not necessary because: test change

### CODE REVIEW

**Comment by emily-shen** [comment]
while we're at it, should fix the request type like here
https://github.com/cloudflare/workers-sdk/blob/01956212410d6cd43419754028b018cfa5039add/packages/create-cloudflare/templates/hello-world/ts/test/index.spec.ts#L8

File: packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { env, ctx, createExecutionContext, waitOnExecutionContext, SELF } from 'cloudflare:test';
+import { env, createExecutionContext, waitOnExecutionContext, SELF } from 'cloudflare:test';
 import { describe, it, expect } from 'vitest';
 import worker from '../src';
```
Comment (emily-shen) [inline packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js:4]: also github won't let me comment on the relevant line, but I don't think we can pass `ctx` into SELF.fetch() on line 19 and 37 anymore.

Comment (petebacondarwin) [reply packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js:4]: Thank you!

**Review by RamIdeas — APPROVED** [review]

File: packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js
Comment (emily-shen) [inline packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js:1]: there's two `await SELF.fetch(request, env, ctx)` 's here that should probably be `await SELF.fetch(request)` like in the ts template?

Comment (petebacondarwin) [reply packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js:1]: Ugh. Sorry for missing those. Thanks for checking

File: packages/create-cloudflare/templates-experimental/hello-world-with-assets/ts/test/index.spec.ts [hunk 2]
```diff
@@ -16,14 +16,14 @@ describe('Hello World user worker', () => {
 
 		it('responds with "Hello, World!" (integration style)', async () => {
 			const request = new Request('http://example.com/message');
-			const response = await SELF.fetch(request, env, ctx);
+			const response = await SELF.fetch(request, env);
```
Comment (petebacondarwin) [inline packages/create-cloudflare/templates-experimental/hello-world-with-assets/ts/test/index.spec.ts:19]: I forgot to remove the `env` here too!

**Pull request merged**